defaults/main.yml

```diff
@@ -12,12 +12,27 @@ frr_daemons:
   ripngd: false
   zebra: true
 
+# Define if we want to use traditional or datacenter FRR defaults
+# If undefined than current FRR default value is used implicitly (traditional)
+#frr_defaults: "datacenter"
```

Comment:
Could we put something more in the readme for this? Not sure how we would go about doing that but might make it easier to follow there too ... like and example of using datacenter frr defaults?

Reply:
I can put there a link to official docs http://docs.frrouting.org/en/latest/basic.html#profiles. Actual values of timers can be found only in source code or by reviewing changes in running-config after applying it.

Reply:
I think that would be best - other than that this looks great! Thanks for all your contributions!